### PR TITLE
ooniprobe 1.3.0 (new formula)

### PR DESCRIPTION
**Pull request by tdsmith**
OONI, the Open Observatory of Network Interference, is a global observation network which aims is to collect high quality data using open methodologies, using Free and Open Source Software (FL/OSS) to share observations and data about the various types, methods, and amounts of network tampering in the world.

cc @hellais

### CODE REVIEW

File: Library/Formula/ooniprobe.rb [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    man1.install Dir["data/*.1"]
+    (share/"ooni").install Dir["data/*"]
+    (var/"lib/ooni").mkpath
+  end
+
+  def caveats; <<-EOS.undent
+    Run `ooniresources --update-inputs --update-geoip` to fetch fresh data
```
Comment (mikemcquaid) [inline Library/Formula/ooniprobe.rb]: Can you run it yourself?

Comment (mikemcquaid) [inline Library/Formula/ooniprobe.rb]: Er, can the formula run it?

Comment (tdsmith) [inline Library/Formula/ooniprobe.rb]: The formula can, sure. It hits the network to fetch some datasets. Is that okay? Figure it makes the most sense in post_install.

Comment (mikemcquaid) [inline Library/Formula/ooniprobe.rb]: Yup, makes sense to me :+1:

**Comment by hellais** [comment]
This is great news!

Thanks for this!

**Pull request closed without merging**